recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "tardis" %}
+{% set version = "0.5.8" %}
+{% set sha256 = "8918af5f4e76ebd78b10aba48ceb5eadb8d544153d4bb746ee8a8c0b23f711bd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/AgResearch/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
```

Comment:
We don't build on windows, but it looks like you want to restrict to py27, so change `# [win]` to `# [not py27]`

Reply:
Thanks, yes.  Done.